web_monitoring/differs.py

```diff
@@ -51,11 +51,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    return web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
 
+def compute_dmp_diff(a_text, b_text, timelimit=4):
+    diff_codes = {'=': 0, '-': -1, '+': 1}
 
-d = diff_match_patch.diff_match_patch()
-
+    if(isinstance(a_text, str) and isinstance(b_text, str)):
```

Comment:
Minor point: as a matter of style, I think we're following guidelines that recommend all `if` and `elif` should use a space before the parentheses; i.e., `if (...)` not `if(...)`. (This applies not just to this line but to all of the code below.)